### DISCUSSION TITLE
Not Connecting Any Longer

### DISCUSSION THREAD

**Discussion by Ulltrra** [fredemmott/StreamDeck-Discord#44] [Q&A] [answered]
I was previously using this plugin with no hiccups but I had to get my PC repaired and after bringing it home and trying to put a new app id and secret, it no longer wants to connect and I am not getting the authorize prompt on discord. When clicking reconnect/ reauthenticate, nothing happens and nothing changes or shows up under the debug information. Any help would be awesome on this. Thank you!

**Comment by fredemmott** [comment, marked as answer]
- try restarting both discord and the streamdeck software
- check everything is copied *exactly* - `https://localhost`, `https://localhost/`, and `http://localhost` are all different
- do not run either discord or the streamdeck software as administrator
